lib/config/error-codes/throw.js

```diff
@@ -68,24 +68,24 @@ function throwError (domain, subdomain, error, ...placeholders) {
 }
 
 /**
- * Wrap getError and throw with the provided domain and subdomain.
+ * Wrap get and throw with the provided domain and subdomain.
 
  * The function have now the following signature:
- *   - getError(error, ...placeholders)
+ *   - get(error, ...placeholders)
  *   - throw(error, ...placeholders)
  *
  * @param  {string} domain - Domain (eg: 'external')
  * @param  {string} subdomain - Subdomain (eg: 'elasticsearch')
  */
 function wrap (domain, subdomain) {
   return {
-    getError: (error, ...placeholders) => getError(domain, subdomain, error, ...placeholders),
+    get: (error, ...placeholders) => getError(domain, subdomain, error, ...placeholders),
     throw: (error, ...placeholders) => throwError(domain, subdomain, error, ...placeholders)
   };
 }
 
 module.exports = {
-  getError,
   wrap,
-  throw: throwError
+  throw: throwError,
+  get: getError,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)